src/en-GB/discover/index.html

```diff
@@ -7,15 +7,193 @@
+        expertise and rapid development, the Ceph community ensures Ceph is, and continues to be, industry leading. Becoming a part of
+        Ceph’s community puts you at the forefront of data storage revolution.
+      </p>
+      <a class="a inline-block mb-8 lg:mb-12" href="/{{ locale }}/community/">Connect with the community</a>
+      <h3 class="h3">Hardware independent</h3>
+      <p class="mb-0 p">
+        Ceph can be deployed on commonly used and widely available hardware, giving you limitless storage scalability with minimal hardware
+        investment, resulting in reduced costs and no vendor lock in. You purchase the hardware you require, or use the hardware you already
+        have, deploy Ceph, and let the software do the rest. Reduce your OPEX through utilising cost-effective infrastructure systems.
+      </p>
+    </div>
+  </div>
+</section>
+
+<hr class="hr" />
 
-  <div class="color-white max-w-160 mx-auto relative text-center z-1">
-    <h2 class="h1 mb-8">Get Started with Ceph</h2>
-    <p class="mb-0 p">
-      Ceph is a unified, distributed storage system designed for excellent performance, reliability and scalability.
-    </p>
+<section class="section">
+  <h2 class="h2 mb-12 lg:mb-16">Driving innovation around the world</h2>
+  <div class="grid lg:grid--cols-2 grid--gap-14 lg:grid--gap-28">
+    <div>
+      <p class="standout">
+        Ceph is the future of enterprise storage, driving innovation, novel data management and digital transformation. From PhD thesis to
+        enterprise ready solution, Ceph has rapidly grown to become a global phenomenon, redefining data storage for enterprises across the
+        world.
+      </p>
+      <div class="flex flex--gap-4 lg:flex--gap-6 flex--wrap mb-4">
+        <img alt="" class="rounded-full w-20" loading="lazy" src="https://via.placeholder.com/50x50" />
+        <img alt="" class="rounded-full w-20" loading="lazy" src="https://via.placeholder.com/50x50" />
+        <img alt="" class="rounded-full w-20" loading="lazy" src="https://via.placeholder.com/50x50" />
+        <img alt="" class="rounded-full w-20" loading="lazy" src="https://via.placeholder.com/50x50" />
+      </div>
+      <a class="a" href="/{{ locale }}/discover/customers/">See who’s using Ceph today</a>
+    </div>
+    <div>
+      <h3 class="h3">Any use case</h3>
+      <p class="p">
+        Organisations that use and store data of any type can use Ceph. From start-up enterprises to academic institutions, the benefits of
+        Ceph ensure intelligent data management and insights, leading to better business decision. For those looking to secure and protect
+        their data, Ceph provides reassurance organisations need, including automatic data redundancy, no single point of failure and
+        self-managing clusters.
+      </p>
+      <a class="a inline-block mb-8 lg:mb-12" href="/{{ locale }}/solutions/case-studies/">Find out how Ceph is used</a>
```

Comment:
```suggestion
      <a class="a inline-block mb-8 lg:mb-12" href="/{{ locale }}/solutions/">Find out how Ceph is used</a>
```

I'd suggest we just link this straight to the Solutions hub.

The `/solutions/case-studies` page is more of a "it exists because it's needed to create a sensible hierarchy" (and to set a precedent for the filtered Case study category pages), but not something that was intended as a place to drive users to, based on the initial wires.